app/views/users/show.html.erb

```diff
@@ -8,4 +8,13 @@
       </h1>
     </section>
   </aside>
+  <div class="col-md-8">
+    <% if @user.microposts.any? %>
+      <h3><%= t "micropost.name" %> (<%= @user.microposts.count %>)</h3>
```

Comment:
count, size, length khác gì nhau

Reply:
Giống: Đều dùng để đếm số lượng phần tử trong mảng, collection(active record)
Khác:

Trong mảng:
count: có thể sử dụng đi kèm điều kiện tính tán để tính số lượng phần tử
length, size: không thể sử dụng đi kèm điều kiện
Trong collection(active record):
count: nó dùng câu lệnh "select count(*) from ..." để lấy ra số lượng bản ghi, count ko được lưu trữ đối tượng mà chúng ta gọi, mỗi lần gọi count thì nó sẽ thực hiện câu truy vấn trên dù trước đó đã gọi hay là chưa
size: khi object đã đuợc load, nó sẽ thực hiện trên chính object đó trong bộ nhớ (không query vào database), còn khi object đó chưa được load, nó sẽ thực hiện câu lệnh (select count(*) from …)
length: giống với size,nếu object chưa đc load nó sẽ nạp toàn bộ object vào bộ nhớ và thực hiện đếm